src/style/main.scss

```diff
@@ -1,10 +1,15 @@
+html {
```

Comment:
Flagging this - as discussed, I'd like to avoid affecting anything outside of `.swagger-ui`.

@Rody-Kirwan, was this included so that the `rem`s  are based at 10px?

Reply:
@shockey Yes. Exactly - However if `rem` is still preferred with this limitation in mind we can use the slightly more awkward 16px standard. Will wait for your advice on this before making any change

Reply:
@Rody-Kirwan, I've tested out a few different scenarios, here's what I propose:

- Migrate to using `em`s instead of `rem`s
- Keep the `font-size: 62.5%;` style, but move it from `html` to `.swagger-ui`

From my point of view, this lets us (a) keep the convenience of `1em === 10px`, (b) stay contained within our own class instead of changing the state of the overall document, and (c) still allow users to scale Swagger UI up and down as needed, just cascade over our `font-size`.

Let me know what you think!

Reply:
> I'll update the relevant internal docs

done!